### PR TITLE
Test/migrate generation script

### PR DESCRIPTION
**Pull request by Reillyhewitson**
## Description
Update the tests so that they find the new data that has been generated due to changes in the generator script included with datagateway-api.

Tests have been changed as little as possible however due to validation python_icat requires some of the date filter tests needed to use todays date instead of the 1st of the month. This is because python_icat will block inserting an arbitrary creation and modification date. 

## Testing instructions
Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking
connect to #{issue number}

### CODE REVIEW

File: packages/datagateway-dataview/package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "e2e:interactive": "start-server-and-test e2e:serve http://localhost:3000 cy:open",
     "e2e": "start-server-and-test e2e:serve http://localhost:3000 cy:run",
     "cy:open": "cypress open",
-    "cy:run": "cypress run",
+    "cy:run": "cypress run --browser 'chrome'",
```
Comment (MRichards99) [inline packages/datagateway-dataview/package.json:68]: Is the `--browser` option needed?

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -151,7 +151,7 @@ jobs:
         with:
           repository: ral-facilities/datagateway-api
           path: datagateway-api
-          ref: v6.0.0
+          ref: feature/icat-backend-generator#369
```
Comment (MRichards99) [inline .github/workflows/ci-build.yml:154]: I know you know but these references will need to change before merging this PR

**Review by MRichards99 — COMMENTED** [review]
Looks all good so far, just wondered if the `--browser` is required?

File: packages/datagateway-dataview/package.json (hunk 1 above)
Comment (Reillyhewitson) [reply packages/datagateway-dataview/package.json]: No, not anymore has been remove.

File: .github/workflows/ci-build.yml (hunk 2 above)
Comment (Reillyhewitson) [reply .github/workflows/ci-build.yml]: These references have been updated in the latest commit.

**Review by MRichards99 — APPROVED** [review]

**Pull request merged**